### PR TITLE
qperf: Require rdma-core and libibmad for IB/RDMA support

### PR DESCRIPTION
**Pull request by ArmedAviator**
This change will enable support measuring RDMA statistics with libibmad and rdma-core.

### CODE REVIEW

File: srcpkgs/qperf/template [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 hostmakedepends="automake"
+makedepends="rdma-core-devel libibmad-devel"
+depends="rdma-core libibmad"
 short_desc="Measure RDMA and TCP/UDP latency and throughput performance"
-maintainer="Rich G <rich@richgannon.net>"
+maintainer="Rich Gannon <rich@richgannon.net>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/linux-rdma/qperf"
-distfiles="https://github.com/linux-rdma/qperf/archive/v${version}.tar.gz"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```
Comment (newbluemoon) [inline srcpkgs/qperf/template]: IMHO you should leave it as it was and not use `${homepage}` here not least for better readability.

**Comment by newbluemoon** [comment]
Needs #9094 first.

File: srcpkgs/qperf/template [hunk 2]
```diff
@@ -1,14 +1,16 @@
 # Template file for 'qperf'
 pkgname=qperf
 version=0.4.11
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake"
+makedepends="rdma-core-devel libibmad-devel"
+depends="rdma-core libibmad"
```
Comment (newbluemoon) [inline srcpkgs/qperf/template]: These should be detected automatically. Check what xbps-src prints out after

`... running pre-pkg hook: 04-generate-runtime-deps ...`

Comment (maxice8) [inline srcpkgs/qperf/template]: stop doing this and read about 'Runtime dependencies' on the manual

File: srcpkgs/qperf/template (hunk 1 above)
Comment (ArmedAviator) [reply srcpkgs/qperf/template]: Fixed.

File: srcpkgs/qperf/template (hunk 2 above)
Comment (ArmedAviator) [reply srcpkgs/qperf/template]: Fixed.  Thanks for the tip.

Comment (ArmedAviator) [reply srcpkgs/qperf/template]: Fixed.

Comment (maxice8) [reply srcpkgs/qperf/template]: Here is the revelant section of our manual https://github.com/void-linux/void-packages/blob/master/Manual.md#deps_runtime

**Pull request closed without merging**